site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,96 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supported storage addons attached to workloads: you need to run 
+`copilot svc deploy` in order to deploy the storage, and the storage is deleted along with the service
+when you run `copilot svc delete`.
+
+Now, you have the option to create environment storage addons: the storage is deployed when you run `copilot env deploy`,
+and isn't deleted until you delete the environment by running `copilot env delete`.
+
+Similar to the workload storage, the environment storage is, under the hood, just another [environment addon](../docs/developing/addons/environment.en.md)!
+
+### [Database-Per-Service](https://docs.aws.amazon.com/prescriptive-guidance/latest/modernization-data-persistence/database-per-service.html) By Default
+In the microservice world, it is generally recommended to set up data storages that are each private to a microservices,
+instead of monolith storages that are shared by multiple services.
+This pattern preserves the core characteristics of microservices - loose coupling.
+Copilot encourages you to follow this database-per-service pattern. By default, a storage resource that Copilot generates
+is assumed to be accessed by one service or job.
+
+!!!note ""
+	However, each product has its own unique situation. If you do need your data storage to be shared by multiple service,
+	you can modify the CloudFormation template that Copilot generates for you to achieve your goal.
+
+Here is an example of prompts that you might see.
+
+!!! note ""
+	```term
+	$ copilot storage init
+    What type of storage would you like to create?
+    > DynamoDB            (NoSQL)
+      S3                  (Objects)
+      Aurora Serverless   (SQL)
+
+    Which workload needs access to the storage? 
+    > api
+      backend
+
+    What would you like to name this DynamoDB Table? movies
+
+    Do you want the storage to be created and deleted with the api service?
+      Yes, the storage should be created and deleted at the same time as api
+    > No, the storage should be created and deleted at the environment level
+	```
+
+You can skip the prompts using the flags. The following command is equivalent to the prompts above:
+```console
+copilot storage init \
+--storage-type "DynamoDB" \
+--workload "api" \
+--name "movies" \
+--lifecycle "environment"
+```
+
+After you've answered all the prompts or skipped them by flags, Copilot will generate the CloudFormation template that defines the DynamoDB storage
+under your "copilot/environments" directory. In addition, it will generate the necessary access policy; here, one that grants "api" service 
```

Comment:
or??
```suggestion
under your `copilot/environments` directory. In addition, it will generate the necessary access policy; here, one that grants "api" service 
```